consensus/src/payload_manager.rs

```diff
@@ -88,13 +86,10 @@ impl PayloadManager {
 
                 let mut tx = coordinator_tx.clone();
 
-                if let Err(e) = tx
-                    .send(CoordinatorCommand::CommitNotification(
-                        block_timestamp,
-                        batches,
-                    ))
-                    .await
-                {
+                if let Err(e) = tx.try_send(CoordinatorCommand::CommitNotification(
+                    block_timestamp,
+                    batches,
+                )) {
                     warn!(
                         "CommitNotification failed. Is the epoch shutting down? error: {}",
```

Comment:
Could we add the block_timestamp and batches to the warn message? try_send failing should be fine, but just in case want to observe this.